django/db/models/expressions.py

```diff
@@ -96,6 +107,24 @@ def __and__(self, other):
             "Use .bitand() and .bitor() for bitwise logical operations."
         )
 
+    def __eq__(self, other):
+        return self._combine(other, self.EQ, False)
+
+    def __ne__(self, other):
+        return self._combine(other, self.NE, False)
+
+    def __lt__(self, other):
+        return self._combine(other, self.LT, False)
+
+    def __le__(self, other):
+        return self._combine(other, self.LE, False)
+
+    def __gt__(self, other):
+        return self._combine(other, self.GT, False)
+
+    def __ge__(self, other):
+        return self._combine(other, self.GT, False)
```

Comment:
Same as @pope1ni mentioned previously, this should be `self.GE`?

Reply:
I still miss one!